<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оценка трудоёмкости разработки интерфейса пользователя</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_sppr">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Редактор параметров СППР</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>501</width>
      <height>294</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Оцените стандартное количество строк кода для реализация элемента:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Название</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>React</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Angular</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Vue.js</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="nameEM_1">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Кнопка:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Меню:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Поле ввода:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_4">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Текстовое поле:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_5">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Ссылка:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_6">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Флаг:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_7">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Радиокнопка:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameEM_8">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Таблица:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLineEdit" name="react_0"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="react_7"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLineEdit" name="angular_0"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="angular_7"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLineEdit" name="vue_0"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="vue_7"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="parameters_save">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>162</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parameters_return">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуть исходные данные</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
